### ISSUE TITLE
JS error after update 2.15.4

### ISSUE THREAD

**Issue by Nookiee** [RocketChat/Rocket.Chat.Electron#1288]
## My Setup
- Operating System: Win 10 Pro (v1809)
- App Version: 2.15.4
- Installation type: .exe

- [X] I have tested with the latest version
- [X] I can simulate the issue easily

## Description
Simply start the Desktop App after updating to 2.15.4
===>
![image](https://user-images.githubusercontent.com/1245343/62756156-ff153d00-ba76-11e9-93a3-0b29e487281f.png)

### Current Behavior
It crashes.

### Expected Behavior
Start the application.

**Comment by volphy** [comment]
Looks like it cannot connect to Bugsnag:
https://www.bugsnag.com/

**Comment by regetskcob** [comment]
@volphy same problem here, was working in the previous version.

**Comment by Nookiee** [comment]
> Looks like it cannot connect to Bugsnag:
> https://www.bugsnag.com/

Obviously, but after reinstalling 2.15.3 it can connect to Bugsnag :) (atleast i dont experience any issues)

**Comment by Malachiasz** [comment]
Same here. App doens't start because of that.

**Comment by renehennig** [comment]
Same here. MacOs Catalina - 10.15 Beta (19A526h)

**Comment by nexidus** [comment]
Same here on Windows 10 1903 Build 18362.267 VM.

**Comment by atemp** [comment]
same problem in CentOS 7

**Comment by jthomae1** [comment]
Some problem on MacOS Mojave 10.14.6.

![Bildschirmfoto 2019-08-09 um 08 29 37](https://user-images.githubusercontent.com/8523176/62758850-da719300-ba7f-11e9-9ee3-1ab8370b2ac3.png)

**Comment by frafu** [comment]
Same on Ubuntu 18.04.3 LTS with snap.

**Comment by mvhirsch** [comment]
Same on Fedora 29.

I think there is a npm package missing in the release-file?

**Comment by infabo** [comment]
Did you try to clear the cache?

**Comment by frafu** [comment]
> Did you try to clear the cache?

I removed and installed the snap packages a few times. So, i guess the cache was deleted.

**Comment by android116** [comment]
Error in openSUSE 15.1:

`$ rocketchat-desktop 
A JavaScript error occurred in the main process
Uncaught Exception:
Error: Cannot find module '@bugsnag/js'
    at Module._resolveFilename (internal/modules/cjs/loader.js:584:15)
    at Function.Module._resolveFilename (/opt/Rocket.Chat/resources/electron.asar/common/reset-search-paths.js:43:12)
    at Function.Module._load (internal/modules/cjs/loader.js:510:25)
    at Module.require (internal/modules/cjs/loader.js:640:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at /opt/Rocket.Chat/resources/app.asar/app/main.js:10:31
    at Object.<anonymous> (/opt/Rocket.Chat/resources/app.asar/app/main.js:1579:3)
    at Object.<anonymous> (/opt/Rocket.Chat/resources/app.asar/app/main.js:1582:3)
    at Module._compile (internal/modules/cjs/loader.js:693:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:704:10)`

**Comment by infabo** [comment]
![image](https://user-images.githubusercontent.com/3999104/62764422-20355800-ba8e-11e9-8e9c-03645a28b6cf.png)

:rofl:

**Comment by frafu** [comment]
It seems like a major fuck up.
How can this happen? What kind of quality assurance is in place?

**Comment by infabo** [comment]
> It seems like a major fuck up.
> How can this happen? What kind of quality assurance is in place?

valid question. but it's still open source - I guess a hotfix-PR may be welcome. :wink:

**Comment by mrwogu** [comment]
I think that `@bugsnag/js` dependency should be moved to dependencies instead of devDependencies cause its using during producation application lifecycle.

package.json
https://github.com/RocketChat/Rocket.Chat.Electron/blob/develop/package.json#L40

errorHandling.js
https://github.com/RocketChat/Rocket.Chat.Electron/blob/develop/src/errorHandling.js#L1

**Comment by mawatech** [comment]
> I think that `@bugsnag/js` dependency should be moved to `dependencies` instead of `devDependencies` cause its using during production application lifecycle.
> 

This seems to fix the error, at least rocket chat starts again!

**Comment by localguru** [comment]
Same with snap on xubuntu disco:

```
:~$ env BAMF_DESKTOP_FILE_HINT=/var/lib/snapd/desktop/applications/rocketchat-desktop_rocketchat-desktop.desktop /snap/bin/rocketchat-desktop
A JavaScript error occurred in the main process
Uncaught Exception:
Error: Cannot find module '@bugsnag/js'
    at Module._resolveFilename (internal/modules/cjs/loader.js:584:15)
    at Function.Module._resolveFilename (/snap/rocketchat-desktop/68/resources/electron.asar/common/reset-search-paths.js:43:12)
    at Function.Module._load (internal/modules/cjs/loader.js:510:25)
    at Module.require (internal/modules/cjs/loader.js:640:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at /snap/rocketchat-desktop/68/resources/app.asar/app/main.js:10:31
    at Object.<anonymous> (/snap/rocketchat-desktop/68/resources/app.asar/app/main.js:1579:3)
    at Object.<anonymous> (/snap/rocketchat-desktop/68/resources/app.asar/app/main.js:1582:3)
    at Module._compile (internal/modules/cjs/loader.js:693:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:704:10)
```

**Comment by mcclint680** [comment]
> Looks like it cannot connect to Bugsnag:
> https://www.bugsnag.com/

Ironic. You'd think Bugsnag would have caught that one. :)

**Comment by ml-** [comment]
@tassoevan the 2.15.5 tag points to a wrong commit.

**Comment by bittner** [comment]
Seems to be [fixed with release 2.15.5](https://github.com/RocketChat/Rocket.Chat.Electron/commit/75846358b80785b57366a6ab7f820394d1ae0d54). The snap on my Ubuntu Bionic box starts up fine.

**Comment by Nookiee** [comment]
Seems to be fixed with the above stated commit.

@tassoevan Thanks!
